extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -23,6 +92,18 @@ export class AmpStoryShoppingAttachment extends AmpStoryPageAttachment {
     this.container_.textContent = this.myText_;
     this.element.appendChild(this.container_);
     applyFillContent(this.container_, /* replacedContent */ true);
+
+    return Promise.resolve(Services.storyStoreServiceForOrNull(this.win)).then(
```

Comment:
Please remove the `Promise.resolve`.
We need to await the `Services.storyStoreServiceForOrNull` so that the buildCallback will only proceed once that resolves.